vehicle/mkvi/software/bms/bms.c

```diff
@@ -37,10 +37,45 @@ void timer0_isr() {
     run_10ms = true;
 }
 
+void timer1_isr() {
+    timer1_cfg.channel_a.interrupt_enable = false;
+    timer1_cfg.channel_b.output_compare_match = 0;
+    timer_init(&timer1_cfg);
+}
+
 void pcint0_callback() {
     bms_core.bspd_current_sense = !!gpio_get_pin(BSPD_CURRENT_THRESH);
 }
 
+static void control_cooling_loop(void) {
+    static int8_t tolerance = 5; // to be defined
```

Comment:
nit: this should be a macro since it doesn't change over time (ie `#define TOLERANCE (5U)`)